src/reducers/index.js

```diff
@@ -1,5 +1,29 @@
+import {
+  SEARCH_USER,
+  UPDATE_USER,
+} from '../actions';
+
 export default function Repos(state = {
   repoOrder: [],
 }, action) {
-  return state;
+  // console.log("IN REDUCER...");
+  // console.log(state);
+  // console.log(action);
+  switch (action) {
+    case SEARCH_USER:
+      // console.log("TESTING SEARCH SPREAD");
+      // console.log(...state);
+      return {
+        ...state,
+        githubUsername: state.githubUsername,
```

Comment:
Did you mean to set the state to its own value? The new value should come in on the action object in some way; maybe on `action.payload`.